build/dependencies.props

```diff
@@ -5,7 +5,9 @@
     <CoreFxVersion>4.4.0-*</CoreFxVersion>
     <DotnetDebToolVersion>2.0.0-preview2-*</DotnetDebToolVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
+    <JsonNetInMSBuildVersion>9.0.1</JsonNetInMSBuildVersion>
     <MoqVersion>4.7.49</MoqVersion>
+    <MsBuildPackageVersions>15.1.1012</MsBuildPackageVersions>
```

Comment:
Doesn't really matter as these are reference assemblies. MSBuild resolves this assembly to whatever version it carried. But since you've pointed it out, we should upgrade the other tools in our org.